src/configs/chalk.c42spec

```diff
@@ -2676,7 +2676,7 @@ validate in the transparency log.
 
   field use_signing_key_backup_service {
     type:    bool
-    default: true
+    default: false
```

Comment:
changing this default to `false` as otherwise OSS chalk shows errors on `chalk setup`:

```
error: auth_config.crashoverride is referenced but its missing in the config
error: Could not retrieve Chalk Data API token from configuration profile. Unable to use Signing Key Backup Service.
error: Could not store password. Either try again later, or use the below password with the CHALK_PASSWORD environment variable. We attempt to store as long as use_signing_key_backup_service is true.
```

to enable it for the console integration, am enabling it in `connect.c4m` as thats what defines `auth_config.crashoverride`

https://github.com/crashappsec/chalk/blob/de311771915d36e712baa36a23e47cd9618a93ef/configs/connect.c4m#L34

cc @viega